### ISSUE TITLE
cannot retrain the networks

### ISSUE THREAD

**Issue by an9236868** [matthewwicker/CertifiableBayesianInference#4]
Hi Matthew, 
Thanks for your effort to release the source code. Unfortunately, I cannot replicate the results by retraining the networks given the current source code. For example, these are from the log file when I trained using the given  `CertifiableBayesianInference/PGD_Experiments` on `MNIST` dataset: 
```
INFO:root:Epoch: 1, Train: [Loss: 2.324, Acc: 0.100], Test: [Loss: 2.303, Acc: 0.101]
INFO:root:Epoch: 2, Train: [Loss: 2.303, Acc: 0.099], Test: [Loss: 2.303, Acc: 0.096]
INFO:root:Epoch: 3, Train: [Loss: 2.303, Acc: 0.099], Test: [Loss: 2.303, Acc: 0.101]
INFO:root:Epoch: 4, Train: [Loss: 2.303, Acc: 0.101], Test: [Loss: 2.303, Acc: 0.103]
INFO:root:Epoch: 5, Train: [Loss: 2.303, Acc: 0.102], Test: [Loss: 2.303, Acc: 0.096]
INFO:root:Epoch: 6, Train: [Loss: 2.303, Acc: 0.100], Test: [Loss: 2.303, Acc: 0.101]
INFO:root:Epoch: 7, Train: [Loss: 2.303, Acc: 0.099], Test: [Loss: 2.303, Acc: 0.096]
INFO:root:Epoch: 8, Train: [Loss: 2.303, Acc: 0.099], Test: [Loss: 2.303, Acc: 0.096]
INFO:root:Epoch: 9, Train: [Loss: 2.303, Acc: 0.101], Test: [Loss: 2.303, Acc: 0.101]
INFO:root:Epoch: 10, Train: [Loss: 2.303, Acc: 0.098], Test: [Loss: 2.303, Acc: 0.096]
```
As we can see that the network didn't learn any things after 10 epochs and the loss is stagnate. Can you please help to double check  and any given details on the versions of `tensorflow` or `keras` that can replicate the results would be appreciated. Thanks!

**Comment by an9236868** [comment]
this is related to the issue #1, I fixed that issue and the model is able to train now
